persistence/src/main/java/edu/unc/lib/dl/persist/services/edit/ExpireEmbargoService.java

```diff
@@ -82,27 +86,20 @@ public void expireEmbargoes() {
                 RepositoryObject repoObj = repoObjLoader.getRepositoryObject(pid);
                 Resource resc = repoObj.getResource();
 
-                String eventText = null;
-                boolean expiredEmbargo = false;
-
                 // remove embargo
                 String embargoDate = resc.getProperty(embargoUntil).getString();
                 repoObjFactory.deleteProperty(repoObj, embargoUntil);
                 pids.add(pid);
-                expiredEmbargo = true;
-
-                if (expiredEmbargo) {
-                    eventText = "Expired an embargo which ended " +
-                            formatDateToUTC(parseUTCToDate(embargoDate));
-                    // Produce the premis event for this embargo
-                    repoObj.getPremisLog().buildEvent(Premis.Dissemination)
-                            .addSoftwareAgent(SoftwareAgent.embargoExpirationService.getFullname())
-                            .addEventDetail(eventText)
-                            .writeAndClose();
-                }
+                String eventText = "Expired an embargo which ended " +
+                        formatDateToUTC(parseUTCToDate(embargoDate));
+                // Produce the premis event for this embargo
+                repoObj.getPremisLog().buildEvent(Premis.Dissemination)
+                        .addSoftwareAgent(SoftwareAgent.embargoExpirationService.getFullname())
+                        .addEventDetail(eventText)
+                        .writeAndClose();
             } catch (Exception e) {
                 tx.cancelAndIgnore();
-                throw e;
+                log.error("Failed to expire embargo: {}", e);
```

Comment:
you don't actually need the `{}` for an exception when its the last parameter in a log statement. But it probably wouldn't be a bad idea to log the PID of the object that it failed to clear the embargo on along with the exception